pubspec.yaml

```diff
@@ -4,7 +4,7 @@ description: General utilities for Dart projects.
 homepage: https://www.github.com/Workiva/w_common
 
 environment:
-  sdk: '>=2.4.0 <3.0.0'
+  sdk: '>=2.7.2 <3.0.0'
```

Comment:
extension methods need >= 2.6.0 or so IIRC. Might not want to bump the min too high, but I figured everyone is likely at least at 2.7.2 (where Workiva is currently stuck).